README.md

```diff
@@ -62,19 +62,23 @@ A Singularity container can be launched in a variety of different ways
 depending on what you wanted to do with it. A simple method might be to
 launch an interactive shell within the container image as follows:
 
+```shell
```

Comment:
Looking at the rendered output of this, I'm not so sure we want to do it - the bash syntax highlighting is rendering part of the prompt in red... since it thinks it's a '>' redirect.